src/Components/dashboard/index.js

```diff
@@ -23,24 +23,27 @@ const Dashboard = () => {
   const [eventToEdit, setEventToEdit] = useState("");
   const [eventsList, setEventList] = useState([]);
   const [refreshEvents, setRefreshEvents] = useState(false);
+  const [lengthEventsList, setLengthEventsList] = useState(0)
+  const [errorJoinEvents, setErrorJoinEvents] = useState(false)
 
-  const pageCount =
-    eventsList && eventsList.length
-      ? Math.ceil(eventsList.length / EVENTS_PER_PAGE)
-      : 0;
-
-  const pagesVisited = pageNumber * EVENTS_PER_PAGE;
-
-  const eventToDraw =
-    eventsList && eventsList.length
-      ? eventsList.slice(pagesVisited, pagesVisited + EVENTS_PER_PAGE)
-      : 0;
+ const pageCount = lengthEventsList ? Math.ceil(lengthEventsList / EVENTS_PER_PAGE) : 0; 
 
   async function getEvents() {
     try {
-      const currentEvents = await getEventsFromServer();
-      setEventList(currentEvents.events);
-      setRefreshEvents(false);
+      const itemsPerPage =  EVENTS_PER_PAGE 
+      const endpoint = `events/${pageNumber}/${itemsPerPage}`
```

Comment:
make sure `pageNumber` and itemsPerPage are numbers, because the endpoint will expect them as number